proto/tendermint/abci/types.proto

```diff
@@ -136,6 +136,11 @@ message RequestPrepareProposal {
   // chain_id is a unique identifier for the blockchain network this proposal
   // belongs to (e.g. mocha-1).
   string chain_id = 3;
+  // height is the height of the proposal block
+  int64 height = 4;
+  // time is the time that will end up in the header. This is the voting power
+  // weighted median of the last commit.
+  google.protobuf.Timestamp time = 5 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
 }
```

Comment:
(Note for myself mainly): Compare with:
 - comet main: https://github.com/cometbft/cometbft/blob/1c6f86c840c64f83f4409ccca6b664234866cc38/proto/tendermint/abci/types.proto#L129-L142
 - v0.36.x: https://github.com/cometbft/cometbft/blob/6321ef2936fbbaf0e3df575982c9485f832a8543/proto/tendermint/abci/types.proto#L105-L118
 - v0.37.x: https://github.com/cometbft/cometbft/blob/be0df2e241f0bd20d51ea2b63e25583ea73dbe21/proto/tendermint/abci/types.proto#L124-L137